storage.go

```diff
@@ -1,35 +1,46 @@
-package dynamodbstorage
+package skydbstorage
 
 import (
 	"context"
 	"encoding/base64"
+	"encoding/json"
 	"errors"
 	"fmt"
-	"log"
+	"strings"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/dynamodb"
-	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/skynetlabs/certmagic-storage-skydb/skydb"
 
-	caddy "github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2"
 	"github.com/caddyserver/certmagic"
+	"go.sia.tech/siad/crypto"
 )
 
 const (
-	contentsAttribute    = "Contents"
-	primaryKeyAttribute  = "PrimaryKey"
-	lastUpdatedAttribute = "LastUpdated"
-	lockTimeoutMinutes   = caddy.Duration(5 * time.Minute)
-	lockPollingInterval  = caddy.Duration(5 * time.Second)
+	lockTimeoutMinutes  = caddy.Duration(5 * time.Minute)
+	lockPollingInterval = caddy.Duration(5 * time.Second)
+
+	// keyListDataKeyString points to a global list of known keys. We need this
+	// list for the List functionality. The list will be updated each time we
+	// Store or Delete a key.
+	keyListDataKeyString = "FvyIax1rlzkpOKsGHGYO4qi/bNgXnjvWFpNXXq13hRc="
+)
+
+var (
+	// The registry doesn't support DELETE as an operation. The next best thing
```

Comment:
This comment is slightly misleading. skyd does have special handling for the case of 34-byte empty skylink when the entry is pointed to by a v2 skylink. This comment seems to imply that the "empty record" is arbitrary.

Reply:
Hm. As we established in this conversation https://github.com/SkynetLabs/certmagic-storage-skydb/pull/1#discussion_r644097646 we're actually writing version 1 skylinks. So that special treatment doesn't seem to matter. 

I will update the comment in order to make it cleat that setting an empty v2 skylink will result in a 404 returned.